cmd/priceFeeder/config/config.toml

```diff
@@ -0,0 +1,79 @@
+[Eth]
```

Comment:
can we simplify this file to use only the relevant config values?
we should have a dedicated config struct